l10n_ar_txt_tucuman/models/account_journal.py

```diff
@@ -0,0 +1,145 @@
+        if line in percepciones:
+            content_datos += '%06.3f' % partner_alicuot.alicuota_percepcion
+        else:
+            content_datos += '%06.3f' % partner_alicuot.alicuota_retencion
+        # 10, MONTO_RET/PER, longitud: 15,2
+        content_datos += '%015.2f' % abs(line.balance)
+        content_datos += '\r\n'
+        return content_datos if content_datos else ''
+
+    def _iibb_tucuman_retper_txt_file(self, line):
+        """ Archivo RETPER.TXT para Tucuman. """
+        content_retper = ''
+        # 1, TIPODOC, longitud: 2
+        content_retper += line.partner_id.l10n_latam_identification_type_id.l10n_ar_afip_code
+        # 2, DOCUMENTO, longitud: 11
+        content_retper += line.partner_id.l10n_ar_vat
+        # 3, NOMBRE, longitud: 40
+        content_retper += line.partner_id.name[:40].ljust(40)
+        # 4, DOMICILIO, longitud: 40
+        content_retper += line.partner_id.street[:40].ljust(40)
+        # 5, Nro, longitud: 5
+        # Hacemos '9' * 5 por el ejemplo que pasó en el archivo adjunto el cliente en la tarea
+        content_retper += '9' * 5
+        # 6, LOCALIDAD, longitud: 15
+        content_retper += line.partner_id.city[:15].ljust(15)
+        # 7, PROVINCIA, longitud: 15
+        content_retper += line.partner_id.state_id.name[:15].ljust(15)
+        # 8, NO USADO, longitud 11
+        content_retper += ' ' * 11
+        # 9, C. POSTAL, longitud: 8
+        content_retper += '%8s' % line.partner_id.zip
+        content_retper += '\r\n'
+        return content_retper if content_retper else ''
+
+    def _iibb_tucuman_ncfact_txt_file(self, line):
+        # Archivo NCFACT.TXT
+        content_ncfact = ''
+        if line.move_type == 'out_refund':
+            nc_document_number_parts = line.move_id._l10n_ar_get_document_number_parts(
+            line.move_id.l10n_latam_document_number, line.l10n_latam_document_type_id.code)
+            # 1, COD. LUGAR EMISION NC, longitud: 4
+            content_ncfact += str(nc_document_number_parts['point_of_sale']).zfill(4)
+            # 2, NUMERO NV, longitud: 8
+            content_ncfact += str(nc_document_number_parts['invoice_number']).zfill(8)
+            # 3, COD LUGAR EMISION FAC, longitud: 4
+            document_number_parts = line.move_id._l10n_ar_get_document_number_parts(
+            line.move_id.reversed_entry_id.l10n_latam_document_number,
+            line.move_id.reversed_entry_id.l10n_latam_document_type_id.code)
+            content_ncfact += str(document_number_parts['point_of_sale']).zfill(4)
+            # 4, NUMERO FAC, longitud: 8
+            content_ncfact += str(document_number_parts['invoice_number']).zfill(8)
+            # 5, TIPO FAC, longitud: 2
+            content_ncfact += line.move_id.reversed_entry_id.l10n_latam_document_type_id.code.zfill(2)
+            content_ncfact += '\r\n'
+            return content_ncfact
+        return ''
```

Comment:
always return at the end of the method
```suggestion
    return content_ncfact
```